dev/AppNotifications/AppNotificationProgressData.h

```diff
@@ -6,7 +6,7 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
     struct AppNotificationProgressData : AppNotificationProgressDataT<AppNotificationProgressData>
     {
         AppNotificationProgressData() = default;
-
+        AppNotificationProgressData(uint32_t sequenceNumber);
```

Comment:
Nit: can we use default parameter here like: AppNotificationProgressData(uint32_t sequenceNumber =1)?

Reply:
This would assume that we provide a default constructor in our projections. However, there is no default projection per API review feedback. We only have an overload ctor.